.travis.yml

```diff
@@ -15,4 +15,7 @@ script:
 branches:
   only:
     - master
+    
+env:
+  - PSO_DIR_PATH=$TRAVIS_BUILD_DIR/scripts/pso/"
```

Comment:
Nit: Quote in the end may not be required

Reply:
Done.